app/src/main/java/woowacourse/shopping/data/repository/RemoteCartProductRepositoryImpl.kt

```
@@ -1,245 +1,79 @@
-
-    override fun getCartProductsInRange(
+class RemoteCartProductRepositoryImpl(
+    private val cartRemoteDataSource: CartRemoteDataSource,
+) : CartProductRepository {
+    override suspend fun insertCartProduct(
+        productId: Long,
+        quantity: Int,
+    ): Long? =
+        try {
+            cartRemoteDataSource.insertProduct(productId = productId, quantity = quantity)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)